nautobot/core/management/commands/generate_test_data.py

```diff
@@ -197,7 +197,7 @@ def _generate_factory_data(self, seed, db_name):
         InterfaceTemplateFactory.create_batch(30, using=db_name)
         PowerPortTemplateFactory.create_batch(30, using=db_name)
         PowerOutletTemplateFactory.create_batch(30, using=db_name)
-        ModuleBayTemplateFactory.create_batch(30, using=db_name)
+        ModuleBayTemplateFactory.create_batch(90, using=db_name)
```

Comment:
I think I needed this for the `DeviceFilterSet.has_empty_module_bays` and `ModuleFilterSet.has_empty_module_bays` tests. I know it was one of the filter tests. All of the components are split between modules and devices and this one is a little different because modules are nestable.

Reply:
Might be able to instead call `ModuleBayTemplateFactory.create_batch()` several times with distinct parameters as an alternative (`create_batch(10, has_parent_device=False)` or whatever). Not critical.